src/contracts/pods/EigenPodManagerStorage.sol

```diff
@@ -75,7 +75,7 @@ abstract contract EigenPodManagerStorage is IEigenPodManager {
      * Likewise, when a withdrawal is completed, this "deficit" is decreased and the withdrawal amount is decreased; We can think of this
      * as the withdrawal "paying off the deficit".
      */
-    mapping(address => int256) public podOwnerShares;
```

Comment:
still need to preserve this public function which scales `podOwnerScaledShares`

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/706/files#diff-e2b069a2e0f6809e6f1cb936e22c81c2218c4d415529adfaa8f33bc32aeb5eb1R291
Yup added here to descale `podOwnerScaledShares` back to `podOwnerShares`